components/definitions/rewards.d.ts

```diff
@@ -116,11 +116,13 @@ declare namespace Rewards {
   export type CaptchaStatus = 'start' | 'wrongPosition' | 'generalError' | 'finished' | null
 
   export enum PromotionTypes {
+    UNKNOWN = -1,
```

Comment:
I'd rather use `UNKNOWN = 0` but `0` is in use and is persisted to disk I believe? 

`UNKNOWN` is needed (or perhaps it should be `INVALID`) as values can be parsed from arbitrary int's in JSON from the server, so if illegal values are sent we drop to UNKNOWN in code below.